docs/_static/js/custom.js

```diff
@@ -0,0 +1,7 @@
+
+
+/** Opens links to external docs in a new tab and links to docs internal to the repo in the current tab. This is to aid navigability. Code taken from https://stackoverflow.com/questions/11716781/open-a-link-in-a-new-window-in-restructuredtext. */
+
```

Comment:
```suggestion
/* 
 * Opens links to external docs in a new tab and links to docs internal to the repo in the current tab.
 * This is to aid navigability. Code taken from:
 * https://stackoverflow.com/questions/11716781/open-a-link-in-a-new-window-in-restructuredtext.
 */
```

Reply:
FYI: If you commit suggestion, you'll need to do a `git pull` to get it locally.

Reply:
I just tried a pull on the branch and it told me everything was up to date.